lms/templates/student_account/account_settings.underscore

```diff
@@ -0,0 +1,25 @@
+<div class="account-settings">
+    <div class="account-settings-header">
+        <div class="account-settings-header-title">
+            <%- gettext("Account Settings") %>
+        </div>
+        <small class="account-settings-header-subtitle"><%- gettext("A collection of all your account settings, including basic personalization controls, demographic information, and linked social accounts.") %></small>
+    </div>
+
+    <div class="account-settings-sections">
+        <% _.each(sections, function(section) { %>
+            <div class="account-settings-section">
+                <div class="account-settings-section-header">
+                    <%- gettext(section.title) %>
+                    <hr>
+                </div>
+                <div class="account-settings-section-body">
+                    <div class="ui-loading-indicator">
```

Comment:
It would be better to have the loading indicator returned in the server response so that it appears immediately. For a slow connection, it will have to wait for all the scripts to download and execute before it will appear. This would mean that the more static elements of the page should be in the Mako template, and Underscore would only be used for the dynamic portion of the page.